azure/template.json

```diff
@@ -230,6 +230,10 @@
                                     "name": "ConfigurationStorageConnectionString",
                                     "value": "[parameters('configurationStorageConnectionString')]"
                                 },
+                                {
+                                    "name": "AzureWebJobsServiceBus",
+                                    "value": "[parameters('nServiceBusConnectionString')]"
+                                },
```

Comment:
```suggestion
```
Can the NServiceBusConnectionString app setting be used in place of this being added?